src/main/java/roomescape/waiting/WaitingWithRank.java

```diff
@@ -0,0 +1,20 @@
+package roomescape.waiting;
+
+public class WaitingWithRank {
+
+    private Waiting waiting;
+    private long rank;
```

Comment:
```suggestion
    private final Waiting waiting;
    private final long rank;
```
final로 선언해도 되겠네요. 추후 record를 학습해보신다면 record로 선언해도 좋을 것 같아요~

Reply:
final을 붙이는건 동의합니다! 하지만, 해당 객체를 record로 사용해도 될까?는 고민이 되는 부분이네요...
현 상황에서는 객체에 아무런 책임이 없어 record로 바꿔도 별 문제 없지만, 추후에 WaitingWithRank 객체만이 가질 수 있는 
로직이 있지 않을까 싶습니다. 단순히 DTO가 아닌, 객체로서의 의미를 가질 수도 있을 것 같아서요...!

Reply:
현재는 책임이 없기 때문에 record로 진행하고, 추후 책임이 부여되게 된다면 객체로 변경하는 방법도 좋을 것 같아요!